qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -353,7 +353,10 @@
 <!-- #samples and #columns -->
 <div class="row">
   <div class="col-md-12">
-    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
+    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.</br>
+    {% if ebi_experiment_accessions %}
+      <b>EBI status: </b> Submitted
```

Comment:
Not really, this is for preps. This is, the study is what's being submitted/updated and the prep is only submitted/not-submitted and we know is submitted if the samples have ebi_experiment_accessions.